common/djangoapps/status/models.py

```diff
@@ -0,0 +1,60 @@
+"""
+Store status messages in the database.
+"""
+
+from django.db import models
+from django.contrib import admin
+from django.core.cache import cache
+
+from xmodule_django.models import CourseKeyField
+
+from config_models.models import ConfigurationModel
+from config_models.admin import ConfigurationModelAdmin
+
+
+class GlobalStatusMessage(ConfigurationModel):
+    """
+    Model that represents the current status message.
+    """
+    message = models.TextField(blank=True, null=True)
+
+    def full_message(self, course_key):
+        """ Returns the full status message, including any course-specific status messages. """
+        cache_key = "status_message.{course_id}".format(course_id=unicode(course_key))
```

Comment:
This is in memcached, which I believe is the default backend for our setup of Django. It's cached for everyone, and the default duration of the cache, according to the [Django documentation](https://docs.djangoproject.com/en/1.4/topics/cache/#cache-arguments) is five minutes.